api/api.go

```diff
@@ -189,6 +192,17 @@ func (api *ManagementAPI) Rename(w http.ResponseWriter, r *http.Request) {
 	return
 }
 
+// Reboot will reboot the device after a delay so a response can be sent back
+func (api *ManagementAPI) Reboot(w http.ResponseWriter, r *http.Request) {
+	go func() {
+		log.Printf("device rebooting in %d seconds", rebootDelaySeconds)
```

Comment:
if you make this a `time.Duration` the "seconds" part doesn't have to go here. Also change the %d to %s.